x11-drivers/nvidia-drivers/metadata.xml

```diff
@@ -4,21 +4,23 @@
 	<maintainer type="person">
 		<email>soap@gentoo.org</email>
 	</maintainer>
-	<longdescription>
-		Nvidia provides different branches of its drivers for different batches of its
-		hardware. When maintaining these ebuilds or other ebuilds depending on them,
-		you will need to keep in mind that the major versions represent these different
-		branches.
-
-		You can use http://www.nvidia.com/Download/Find.aspx to figure out which branch
-		best matches your hardware.
-	</longdescription>
+	<maintainer type="person">
+		<email>sudinave@gmail.com</email>
+		<name>Ionen Wolkens</name>
+	</maintainer>
 	<use>
-		<flag name="X">Install the X.org driver, OpenGL libraries, XvMC libraries, and VDPAU libraries</flag>
 		<flag name="compat">Install non-GLVND libGL for backwards compatibility</flag>
-		<flag name="driver">Install the kernel driver module</flag>
+		<flag name="driver">Install kernel driver modules</flag>
 		<flag name="kms">Enable support for kernel mode setting (KMS)</flag>
 		<flag name="tools">Install additional tools such as nvidia-settings</flag>
 		<flag name="uvm">Install the Unified Memory kernel module (nvidia-uvm) for sharing memory between CPU and GPU in CUDA programs</flag>
```

Comment:
if UVM is now always included, shouldn't this flag be removed?

Reply:
Can't remove until cleanup old, those versions are stable right now.

Reply:
Yes, I mentioned this in my opening comment. The CI is failing because of this too.